kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -930,7 +930,7 @@ private void handleListOffsetRequestV0(KafkaHeaderAndRequest listOffset,
 
         // in v0, the iterator is offsetData,
         // in v1, the iterator is partitionTimestamps,
-        log.warn("received a v0 listOffset: {}", request.toString(true));
+        log.debug("received a v0 listOffset: {}", request.toString(true));
```

Comment:
The same as I said before.

Reply:
By the way, if a new PR was pushed to change log level, `log.debug` should be wrapped in a `if (log.isDebugEnabled())` block. And

Reply:
@BewareMyPower  there is a UT failure, do you know why it's caused by this PR?

Reply:
I think the failed UT is caused by no consumer `backlog ` metrics if using `resetCursorAsync`.
 we should think about how to solve the consumer metrics problem at the same time.